spring-cloud-stream/src/main/java/org/springframework/cloud/stream/adapter/ChannelBindingAdapter.java

```diff
@@ -64,7 +64,7 @@
 
 	private static Logger logger = LoggerFactory.getLogger(ChannelBindingAdapter.class);
 
-	private MessageBus messageBus;
+	private Binder binder;
```

Comment:
Binder is parameterized, so we should consider updating that in the using classes.